### PR TITLE
Fix issue #125 timer/schedule/expression/annotated/Client.java#dayOfW…

### PR DESCRIPTION
**Pull request by chengfang**
…eekOverDayOfMonth failed at midnight

Signed-off-by: Cheng Fang <cfang@redhat.com>

### CODE REVIEW

File: src/com/sun/ts/tests/ejb30/timer/schedule/expression/annotated/ScheduleBean.java [hunk 1]
```diff
@@ -217,10 +217,10 @@ public String attributeDefaults() {
   public String dayOfWeekOverDayOfMonth() {
     TimerInfo info = new TimerInfo("dayOfWeekOverDayOfMonth");
     Calendar cal = Calendar.getInstance();
-    cal.add(Calendar.DAY_OF_WEEK, 1);
+    cal.add(Calendar.DAY_OF_WEEK, 2);
     int nextDayOfWeek = TimerUtil.getForSchedule(Calendar.DAY_OF_WEEK, cal);
     // every year, every month, every hour, every minute, every second, but not
-    // every day. Only for the next day of week. So the timeout method for
+    // every day. Only for 2+ day of week. So the timeout method for
```
Comment (scottmarlow) [inline src/com/sun/ts/tests/ejb30/timer/schedule/expression/annotated/ScheduleBean.java]: What is wrong currently?

Why do we need to make this change?

Thanks,
Scott

**Comment by chengfang** [comment]
The test creates a timer that expires any time next day, and expects there will be no timeout event. When it is run near midnight (11:59:xx PM), the next day is only a second or less away. By the time the test checks for any timeout, the timer already fired since it's already the next day.

So my fix is to change the next day to next next day to make sure it will not expire as expected by the test.  See https://github.com/eclipse-ee4j/jakartaee-tck/issues/125 for more details, and logs showing the failed execution time is always 11:59:xx PM.

**Comment by scottmarlow** [comment]
Awesome, thanks for the explanation Cheng!  :)

**Comment by scottmarlow** [comment]
After we have converted the Jakarta EE TCK master branch to use the Jakarta namespace, we can merge this change.

**Comment by scottmarlow** [comment]
Thanks @chengfang

**Pull request merged**